app/templates/index.html

```diff
@@ -1,35 +1,30 @@
 {% extends "base.html" %}
 {%- from 'govuk_frontend_jinja/components/inset-text/macro.html' import govukInsetText -%}
 {%- from "govuk_frontend_jinja/components/button/macro.html" import govukButton -%}
-
 {% block content %}
-    <div class="govuk-phase-banner flex-parent-element">
-        <p></p>
     </div>
+    <div class="fsd-login-banner-background">
     </div>
-  <div class="govuk-width-container">
-      <h1 class="govuk-heading-xl">Assessment Hub</h1>
-
-      {% if not g.is_authenticated %}
-        <p class="govuk-body">
-            Welcome to the assessment service.
-            <br /><br />Please sign in to continue.
```

Comment:
Looks good but this line is causing an error in tests:

https://github.com/communitiesuk/funding-service-design-assessment/actions/runs/3845296662/jobs/6549283378#step:6:47

Reply:
Could probably just remove this line: https://github.com/communitiesuk/funding-service-design-assessment/blob/0357ea250422ee649d4ba35762350d825314b12b/tests/test_authorisation.py#L71